### ISSUE TITLE
Command line issue with Windows / pipenv

### ISSUE THREAD

**Issue by rcolle** [microsoft/pyright#152]
**Describe the bug**
I'm running on a 3.7 pipenv setup on Windows.
The pyright extension for VS code works perfectly (thanks!), but the command line version fails because of what looks like an issue with locating typeshed type stubs.

I've managed to work around the issue by forcing the typeshed path to the relevant directory in the node_module directory. The following command works perfectly:

```
$ pipenv run yarn run pyright -t node_modules/pyright/dist/typeshed-fallback/
yarn run v1.13.0
$ pyright --project pyrightconfig.json -t node_modules/pyright/dist/typeshed-fallback/
Loading configuration file at C:\views\views\chouket1\workspace\chouket\pyrightconfig.json
Searching for source files
Found 28 source files
0 errors, 0 warnings
```

However, without forcing the typeshed path, I get the following verbose output:
```
$ pipenv run yarn run pyright
yarn run v1.13.0
$ pyright --project pyrightconfig.json --verbose
Loading configuration file at C:\views\views\chouket1\workspace\chouket\pyrightconfig.json
Search paths found for configured python interpreter:
  C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib
  C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\Scripts
  C:\Users\r.colle\AppData\Local\Programs\Python\Python37\Lib
  C:\Users\r.colle\AppData\Local\Programs\Python\Python37\DLLs
  C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur
  C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\site-packages
  c:\views\views\chouket1\workspace\chouket\python
When attempting to get search paths from python interpreter:
  Executing python interpreter
  Skipping 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\Scripts\python37.zip' because it is not a valid directory
  Skipping 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\DLLs' because it is not a valid directory
  Received 7 paths from interpreter
    C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib
    C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\Scripts
    C:\Users\r.colle\AppData\Local\Programs\Python\Python37\Lib
    C:\Users\r.colle\AppData\Local\Programs\Python\Python37\DLLs
    C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur
    C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\site-packages
    c:\views\views\chouket1\workspace\chouket\python
Searching for source files
Found 28 source files
Could not import 'builtins' in file 'C:\views\views\chouket1\workspace\chouket\python\pyket\__init__.py'
  Finding python search paths
  Looking for typeshed stdlib path
  Looking in root directory of execution environment 'C:\views\views\chouket1\workspace\chouket'
  Attempting to resolve using root path 'C:\views\views\chouket1\workspace\chouket'
  Could not find directory 'C:\views\views\chouket1\workspace\chouket\builtins'
  Did not find file 'C:\views\views\chouket1\workspace\chouket\builtins.pyi' or 'C:\views\views\chouket1\workspace\chouket\builtins.py'
  Looking for typeshed path
  Looking for typeshed third_party path
  Looking in python search path 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib'
  Attempting to resolve using root path 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib'
  Could not find directory 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\builtins'
  Did not find file 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\builtins.pyi' or 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\builtins.py'
  Looking in python search path 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\Scripts'
  Attempting to resolve using root path 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\Scripts'
  Could not find directory 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\Scripts\builtins'
  Did not find file 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\Scripts\builtins.pyi' or 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\Scripts\builtins.py'
  Looking in python search path 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\Lib'
  Attempting to resolve using root path 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\Lib'
  Could not find directory 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\Lib\builtins'
  Did not find file 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\Lib\builtins.pyi' or 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\Lib\builtins.py'
  Looking in python search path 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\DLLs'
  Attempting to resolve using root path 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\DLLs'
  Could not find directory 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\DLLs\builtins'
  Did not find file 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\DLLs\builtins.pyi' or 'C:\Users\r.colle\AppData\Local\Programs\Python\Python37\DLLs\builtins.py'
  Looking in python search path 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur'
  Attempting to resolve using root path 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur'
  Could not find directory 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\builtins'
  Did not find file 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\builtins.pyi' or 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\builtins.py'
  Looking in python search path 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\site-packages'
  Attempting to resolve using root path 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\site-packages'
  Could not find directory 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\site-packages\builtins'
  Did not find file 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\site-packages\builtins.pyi' or 'C:\Users\r.colle\.virtualenvs\chouket-T7Dvucur\lib\site-packages\builtins.py'
  Looking in python search path 'c:\views\views\chouket1\workspace\chouket\python'
  Attempting to resolve using root path 'c:\views\views\chouket1\workspace\chouket\python'
  Could not find directory 'c:\views\views\chouket1\workspace\chouket\python\builtins'
  Did not find file 'c:\views\views\chouket1\workspace\chouket\python\builtins.pyi' or 'c:\views\views\chouket1\workspace\chouket\python\builtins.py'
[...]
C:\views\views\chouket1\workspace\chouket\python\pyket\activepivot_process.py
  7:8 - error: Import 'time' could not be resolved
  34:30 - error: 'int' is not defined
  43:9 - error: 'print' is not defined
  57:47 - error: 'int' is not defined
  57:68 - error: 'int' is not defined
  69:19 - error: 'FileNotFoundError' is not defined
  79:16 - error: 'Exception' is not defined
  86:60 - error: 'int' is not defined
  112:21 - error: 'str' is not defined
  115:53 - error: 'open' is not defined
  118:43 - error: 'int' is not defined
  126:18 - error: 'range' is not defined
  129:29 - error: 'open' is not defined
  134:32 - error: 'int' is not defined
  135:20 - error: 'FileNotFoundError' is not defined
  145:15 - error: 'Exception' is not defined
[...]
```

**To Reproduce**
`$ pipenv run yarn run pyright`

**Expected behavior**
It should work

**Screenshots or Code**

**VS Code extension or command-line**
Command-line:
```
$ pipenv run yarn run pyright --version
yarn run v1.13.0
$ pyright --project pyrightconfig.json --version
pyright 1.0.39
```

**Additional context**
Windows 10, Python 3.7.3, pipenv

**Comment by erictraut** [comment]
I have a theory about why this occurs, but I'm not able to confirm because I don't have ready access to my windows machine this week. I've made a change that will be in the next version of pyright. Please verify whether or not my fix worked.

**Comment by erictraut** [comment]
I've attempted a fix in 1.0.42, which I just published. Please try it out and let me know if it works. If not, re-open the bug and I'll diagnose it once I have direct access to a Windows machine.

**Comment by rcolle** [comment]
Works like a charm now.
Thanks for the quick fix!
